styles/base/_links.scss

```diff
@@ -72,20 +72,27 @@
 
   &:hover,
   &:active,
-  &:focus,
-  &:visited {
+  &:focus {
     border-bottom: $sprk-link-has-icon-hover-border-bottom;
     color: $sprk-link-has-icon-hover-color-text;
   }
+
+  &:visited {
+    border-bottom: $sprk-link-has-icon-visited-border-bottom;
+    color: $sprk-link-has-icon-visited-color-text;
+  }
 }
 
 .sprk-b-Link--has-icon .sprk-c-Icon {
   color: $sprk-link-has-icon-color-icon;
   fill: $sprk-link-has-icon-fill;
   stroke: $sprk-link-has-icon-stroke;
+  transition: $sprk-link-transition;
```

Comment:
This would be $sprk-link-has-icon-transition to follow all the others here and it would take the value of $sprk-link-transition. @bobwise  @RVMendoza

Reply:
Fixed in this PR: https://github.com/sparkdesignsystem/spark-design-system/pull/3949